aws/rds/variables.tf

```diff
@@ -1,171 +1,231 @@
+  type        = string
 }
 
-variable "resources_prefix" {
-  description = "Creates resources with this prefix."
+variable "instance_class" {
+  description = "Instance class for the primary DB."
   type        = string
 }
 
-variable "vpc" {
-  type = object({
-    id      = string
-    cidr    = string
-    subnets = optional(list(string), [])
-  })
+# variable "instance_class_replica" {
+#   description = "Instance class for the replica DB."
+#   type        = string
+#   default     = null
+# }
+
+variable "storage_type" {
+  description = "Storage type for the DB."
+  type        = string
+}
+
+variable "engine" {
+  description = "Database engine."
+  type        = string
+}
+
+variable "engine_version" {
+  description = "Version of the database engine."
+  type        = string
+}
+
+variable "username" {
+  description = "Username for the DB."
+  type        = string
+}
+
+variable "password" {
+  description = "Password for the DB."
+  type        = string
 }
 
 variable "allocated_storage" {
-  description = "Allocated DB storage in GiB."
+  description = "Allocated storage for the DB."
   type        = number
-  default     = 10
 }
 
-variable "instance_class" {
-  # See https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.DBInstanceClass.html
-  description = "DB instance class."
+variable "final_snapshot_identifier" {
+  description = "Identifier for the final snapshot."
   type        = string
-  default     = "db.t2.micro"
 }
 
-variable "tags" {
-  type    = map(string)
-  default = {}
+variable "deletion_protection" {
+  description = "Enable deletion protection."
+  type        = bool
+  default     = false
 }
 
-variable "storage_type" {
-  type    = string
-  default = "gp2"
+variable "create_monitoring_role" {
+  description = "Create a monitoring role."
+  type        = bool
+  default     = true
 }
 
-variable "engine" {
-  type    = string
-  default = "postgres"
+variable "monitoring_role" {
+  description = "ARN of the monitoring role."
+  type        = string
 }
 
-variable "engine_version" {
-  type    = string
-  default = "11.6"
+variable "multi_az" {
+  description = "Enable Multi-AZ deployment."
+  type        = bool
+  default     = false
 }
 
-variable "backup_retention_period" {
-  description = "Backup retention period in days."
+variable "max_allocated_storage" {
+  description = "Maximum allocated storage."
   type        = number
-  default     = 30
 }
 
-variable "snapshot_identifier" {
-  description = "Specify a snapshot to create the DB instance from. Triggers recreation."
+variable "vpc" {
+  type = object({
+    id      = string
+    cidr    = string
+    subnets = optional(list(string), [])
+  })
+}
+
+variable "tags" {
+  description = "Tags for the DB."
+  type        = map(string)
+}
+
+variable "performance_insights_enabled" {
+  description = "Enable Performance Insights."
+  type        = bool
+  default     = false
+}
+
+variable "manage_master_user_password" {
```

Comment:
I'd leave the explaining comment on this variable, they pointed us to where we could find more info on this variable in particular